watertap/costing/watertap_costing_package.py

```diff
@@ -61,23 +61,63 @@ def add_LCOW(self, flow_rate, name="LCOW"):
             name (optional) - name for the LCOW variable (default: LCOW)
         """
 
+        denominator = (
+            pyo.units.convert(flow_rate, to_units=pyo.units.m**3 / self.base_period)
+            * self.utilization_factor
+        )
+
         self.add_component(
             name,
             pyo.Expression(
                 expr=(
                     self.total_capital_cost * self.capital_recovery_factor
                     + self.total_operating_cost
                 )
-                / (
-                    pyo.units.convert(
-                        flow_rate, to_units=pyo.units.m**3 / self.base_period
-                    )
-                    * self.utilization_factor
-                ),
+                / denominator,
                 doc=f"Levelized Cost of Water based on flow {flow_rate.name}",
             ),
         )
 
+        c_units = self.base_currency
+        t_units = self.base_period
+        unit_lcows = pyo.Expression(
+            pyo.Any,
+            doc=f"Levelized Cost of Water per unit based on flow {flow_rate.name}",
+        )
+        self.add_component(name + "_per_unit", unit_lcows)
+        for u in self._registered_unit_costing:
+            numerator = 0
+            if hasattr(u, "capital_cost"):
+                capital_cost = pyo.units.convert(u.capital_cost, to_units=c_units)
+                # total capital costs w/ recovery factor
+                numerator += self.capital_recovery_factor * (
```

Comment:
Rather than add the capex and opex components to the numerator (which could be one level of aggregation), the more typical/informative convention for LCOW breakdown is to separately report the capex and opex components separately for each unit. I think summing the fixed and variable opex would be OK.

Reply:
What should the names be?

We can add unlimited expressions will little additional cost. I.e., we could do
`LCOW_unit_total[ unit_name ]`, `LCOW_unit_capex[ unit_name ]`, `LCOW_unit_opex[ unit_name ]`

Reply:
That's a good question. Maybe something like

LCOW_capex[unit_name_or_indirect_capex_component]

LCOW_opex[unitname_or_flow_name]

So LCOW_opex["electricity"] or LCOW_opex["reverse_osmosis"] would work.

Reply:
I'm worried about name clashes if I put units and flows in the same object -- there are other checks to make sure unit names and flow names don't clash with each other. It would probably be best to keep them separate.

Note that flows, by definition, will only have an opex anyways.